src/app/app.router.ts

```diff
@@ -0,0 +1,15 @@
+import {ModuleWithProviders} from '@angular/core';
+import {Routes, RouterModule} from '@angular/router';
+
+export const ROUTER: Routes = [
+  {
+    path: '',
+    loadChildren: 'app/home/home.module#HomeModule'
```

Comment:
Ok, I see that you decided to isolate the `HomeModule`. In that case, your app will always hit the  root of the application on startup, so It's better to preload it too.

Add a [preloading strategy](https://github.com/Centroida/ngx-training-intro/tree/master/Angular4/preloading-strategy) in order to complete this change.